.github/workflows/welcome-first-time-contrib.yml

```
@@ -16,8 +16,11 @@ jobs:
     name: Post welcome message
     if: ${{ !contains(fromJson('["asyncapi-bot", "dependabot[bot]", "dependabot-preview[bot]", "allcontributors[bot]"]'), github.actor) }}
     runs-on: ubuntu-latest
+    permissions:
+      issues: write  # createComment on issues
+      pull_requests: write # createReview on PRs
```

actionlint (1.7.12)

[error] unknown permission scope "pull_requests". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)